localization/english/bookmark/bookmark_603_l_english.yml

```diff
@@ -1,11 +1,39 @@
+ 
+ What destiny awaits the elven king and his people ? To prosper secluded in their magical kingdom ? To unite with their formers allies against a darker threat ? To fall into oblivion alone...Or perhaps to take an even greater place in the world ? "
+
+ #Ner'zhul
+ nerzhul_character:0 "Ner'zhul, the Lich King"
+ nerzhul_character_desc:0 "Once, Ner'Zhul was an orc. As chieftain of the Shadowmoon clan and elder shaman, he was the spiritual leader of the orcs, deceived to unite them into the warmongering Horde by the demon-lord Kil'Jaeden as a tool of war for the Burning Legion. When he tried to oppose his demonic master, Ner'Zhul was betrayed and replaced by his apprentice, Gul'dan, and when the Horde was finally defeated and Draenor destroyed, Ner'Zhul tried to flee, only to end in the claws of the Legion's minions. For his treason, he was remade into a spectral being, the Lich-King, a new weapon of war against Azeroth. Sent to the continent of Northrend to prepare for his mission, the Lich-King discovered his immense powers and unleashed them under the vigilant supervisions of Kil'Jaeden's agents. Ner'Zhul has built the base of a new order, to the Legion, he will be the vanguard of their invasion, and to the people of Azeroth, a god of death about to make a new world reborn in undeath.
+ 
+ Will Ner'Zhul succeed in his task to ensure the Legion's return ? Will he be stopped before such tragedy could occur...Or will he achieve a different destiny, as his own master, and ruler of both the deads and the world ?"
+
+ #Daelin Proudmoore
+ daelin_character:0 "Admiral Daelin"
+ daelin_character_desc:0 "Patriarch of one of Kul Tiras's great houses, lord of Tiragarde and commander of the fleet, Daelin Proudmoore was elected by his peers as Lord-Admiral, ruler of the insular nation. Under Proudmoore's leadership, the merchant kingdom became the strongest naval power in the world, and pledged its fleet to the service of the Alliance. As Grand Admiral of the Alliance's fleet, Daelin Proudmoore brought victory against the Horde during the Second War, during which his son and heir Derek died to orcish hands, and remained one of the Alliance's strongest supporter after its end. Even if the sea protects his kingdom from threats, being epidemic or orc agression, the admiral remains vigilant and knows that his fleet will be vital to the Alliance in case of a new war, he also knows that whoever hold the sea...holds the world.
```

Comment:
```suggestion
daelin_character_desc:0 "Patriarch of one of Kul Tiras's great houses, lord of Tiragarde, and commander of the fleet, Daelin Proudmoore was elected by his peers as Lord Admiral, ruler of the insular nation. Under Proudmoore's leadership, the merchant kingdom became the strongest naval power in the world and pledged its fleet to the service of the Alliance. As Grand Admiral of the Alliance's fleet, Daelin Proudmoore brought victory against the Horde during the Second War, during which his son and heir Derek died to orcish hands, and remained one of the Alliance's strongest supporters after its end. Even if the sea protects his kingdom from threats, being epidemic or orc aggression, the admiral remains vigilant and knows that his fleet will be vital to the Alliance in case of a new war, he also knows that whoever holds the sea... holds the world.
```